agent/src/main/java/com/code_intelligence/jazzer/autofuzz/FuzzTarget.java

```diff
@@ -122,19 +123,20 @@ public static void fuzzerInitialize(String[] args) {
       }
       System.exit(1);
     }
-    List<Class<?>> alwaysIgnore =
-        Arrays.stream(args)
-            .skip(1)
-            .map(name -> {
-              try {
-                return ClassLoader.getSystemClassLoader().loadClass(name);
-              } catch (ClassNotFoundException e) {
-                System.err.printf("Failed to find class '%s' specified in --autofuzz_ignore", name);
-                System.exit(1);
-              }
-              throw new Error("Not reached");
-            })
-            .collect(Collectors.toList());
+
+    Set<Class<?>> alwaysIgnore = new HashSet<>();
+    ClassGraph classGraph = new ClassGraph().enableClassInfo().enableSystemJarsAndModules();
+    try (ScanResult result = classGraph.scan()) {
+      List<Class<?>> allExceptions = result.getSubclasses(Throwable.class).loadClasses();
```

Comment:
I think we could get away with loading only the classes that match the pattern.

Reply:
Changed based on our discussion to a simpler globbing logic